projects/eslint-config/plugins/portal/tests/lib/rules/no-metal-plugins.js

```diff
@@ -62,6 +98,24 @@ ruleTester.run('no-metal-plugins', rule, {
 				},
 			],
 		},
+		{
+			code: "import {MultiMap} from 'metal-multimap';",
```

Comment:
Another package that [doesn't even have this named export](https://github.com/metal/metal-multimap/blob/f1149882dbfda3c1d082a61a2cc756ac1c85a2af/src/MultiMap.js#L141).